### ISSUE TITLE
Problems Running QuPath 0.2.0 on Manjaro Linux (KDE Plasma)

### ISSUE THREAD

**Issue by evan-bailey** [qupath/qupath#369]
Manjaro KDE Edition, Kernel 4.19.69-1-MANJARO, Intel i7-4790, 16GB RAM, 

Extracted from tar.xz using `tar -xf `into /usr/local/src/ and tried to run app as sudo using `./QuPath-0.2.0-m4`.  App starts up but the start dialogue box ("Welcome to QuPath v0.2.0-m4!") will not close.  Close ('x') button and minimize are displaying in the window but not the maximise button and these buttons are unresponsive on both the welcome window and the main program window.  

Also tried extracting and running from GUI (Dolphin) but same issue occurred. 

No menu items are accessible and the windows are non-responsive.  Cannot drag or drop items into QuPath to open images. Simply frozen on the initial x2 windows of the application.  From terminal output may be an issue with a gtk module?

Pasting Terminal Output from `sudo ./QuPath-0.2.0-m4` command:

```
10:11:46.562 [main] [INFO ] qupath.QuPath - Launching QuPath with args: 
Gtk-Message: 10:11:46.750: Failed to load module "appmenu-gtk-module"
10:11:46.955 [JavaFX Application Thread] [INFO ] qupath.lib.gui.prefs.PathPrefs - Locale FORMAT set to en_US
10:11:46.956 [JavaFX Application Thread] [INFO ] qupath.lib.gui.prefs.PathPrefs - Locale DISPLAY set to en_US
10:11:47.045 [JavaFX Application Thread] [WARN ] qupath.lib.gui.QuPathGUI - No directory set for log files! None will be written.
10:11:47.048 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - QuPath build: Version: 0.2.0-m4
Build time: 2019-08-20, 20:38
Latest commit tag: 'e9b60579'
10:11:47.051 [JavaFX Application Thread] [INFO ] qupath.lib.gui.prefs.PathPrefs - Setting tile cache size to 1996.50 MB (25.0% max memory)

(QuPath-0.2.0-m4:17581): Gdk-WARNING **: 10:11:47.691: XSetErrorHandler() called with a GDK error trap pushed. Don't do that.
10:11:48.207 [JavaFX Application Thread] [INFO ] q.l.i.s.b.BioFormatsOptionsExtension - Bio-Formats version 6.2.0
10:11:48.216 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension Bio-Formats server options (Bio-Formats 6.2.0) (20 ms)
10:11:48.235 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension Experimental commands (19 ms)
10:11:48.278 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension ImageJ extension (42 ms)
10:11:48.290 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension JPen extension (12 ms)
10:11:48.294 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension OpenCV extensions (3 ms)
Oct 13, 2019 10:11:48 AM jpen.provider.NativeLibraryLoader$4 run
INFO: loading JPen 2-150301 JNI library: jpen-2-4-x86_64 ...
Oct 13, 2019 10:11:48 AM jpen.provider.NativeLibraryLoader$4 run
INFO: jpen-2-4-x86_64 loaded
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.codehaus.groovy.vmplugin.v7.Java7$1 (file:/usr/local/src/QuPath-0.2.0-m4/app/groovy-2.5.7.jar) to constructor java.lang.invoke.MethodHandles$Lookup(java.lang.Class,int)
WARNING: Please consider reporting this to the maintainers of org.codehaus.groovy.vmplugin.v7.Java7$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
10:11:48.594 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Loaded extension Rich script editor extension (300 ms)
10:11:48.603 [JavaFX Application Thread] [INFO ] q.l.i.s.o.OpenslideServerBuilder - OpenSlide version 3.4.1
10:11:48.698 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Selected style: null
10:11:48.698 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathGUI - Performing update check...
10:11:48.704 [JavaFX Application Thread] [INFO ] qupath.lib.gui.QuPathApp - Starting QuPath with parameters: []

(QuPath-0.2.0-m4:17581): Gdk-WARNING **: 10:11:48.706: XSetErrorHandler() called with a GDK error trap pushed. Don't do that.
```

**Comment by petebankhead** [comment]
The issue looks to be the same as https://github.com/qupath/qupath/issues/336, in which case the switch to use OpenJFX 13 in the next release of QuPath will hopefully resolve it.

If you want to try building it from the `dev-m5` branch (some guidance [here](https://petebankhead.github.io/qupath/2019/10/01/building-qupath.html)) please do let me know if it does work or not. Otherwise I hope v0.2.0-m5 will be available within the next week.

**Comment by petebankhead** [comment]
I'll close this issue, would be good to continue any discussion at #336 
I hope that it's resolved in [v0.2.0-m5](https://github.com/qupath/qupath/releases/tag/v0.2.0-m5) through the update in the JavaFX dependency, but I haven't got a way to test it myself.
